model/src/main/resources/db-update/20030-from-2.2.0-to-2.3.0.sql

```diff
@@ -20,4 +20,7 @@
 BEGIN transaction;
     ALTER TABLE buildrecord ADD COLUMN alignmentpreference varchar(255);
     ALTER TABLE buildconfigsetrecord ADD COLUMN alignmentpreference varchar(255);
+
+    UPDATE buildrecord SET alignmentpreference = 'PREFER_TEMPORARY' WHERE temporarybuild IS TRUE AND alignmentpreference IS NULL;
+    UPDATE buildconfigsetrecord SET alignmentpreference = 'PREFER_TEMPORARY' WHERE temporarybuild IS TRUE AND alignmentpreference IS NULL;
```

Comment:
BTW why do we have it as field in buildRecord and not build attribute?

Reply:
We stored it similarly to the temporary build option, which is also easier to consume.